config/routes.rb

```diff
@@ -1,9 +1,14 @@
 Rails.application.routes.draw do
+  get 'ads/index'
+
+  devise_for :users
+
+  resources :ads
```

Comment:
since you are using only 3 actions from all resources, you might want to narrow them down just not to generate routes you are not really using. 
`resources :ads, only: %i(index new create)`